index.html

```diff
@@ -11,13 +11,77 @@
 
 <body>
 
-  <p class="manifesto"><em class="word" id="what">XXXX</em> que se organiza de forma <em class="word" id="how">XXXX</em> para <em class="word" id="goal">XXXX</em> interfaces.</p>
+  <p
+    id="manifesto" 
+    class="manifesto"
+  >
+    <em class="word" id="what">XXXX</em> que se organiza de forma 
+    <em class="word" id="how">XXXX</em> para <em class="word" id="goal">XXXX</em> interfaces.
+  </p>
 
   <nav class="links">
-    <a href="">facebook</a>
-    <a href="">slack</a>
-    <a href="">blog</a>
-    <a href="">github</a>
+    <div
+      id="facebook-link"
+      class="nav-item"
+    >
+      <img
+        class="img-content"54
+        src="./assets/icons/facebook.png"
```

Comment:
When using PNG files for icons computers with higher pixel density (such as macbooks or smartphones) will render them pixelized. Idealy, we should be using SVG files for icons or something like [Fontawesome](https://fontawesome.com/).

Reply:
I've just fixed that
I just have to say that [Fontawesome](https://fontawesome.com/) doesn't contain slack and blog icons, so I adapted with some icons available on the lib.

![image](https://user-images.githubusercontent.com/43852963/69987921-52783300-151f-11ea-8109-2e437b7123c7.png)